modules/aws-limits.adoc

```diff
@@ -47,10 +47,10 @@ To use the `us-east-1` region, you must increase the EIP limit for your account.
 |5 VPCs per region
 |Each cluster creates its own VPC.
 
-|Elastic Load Balancing (ELB/NLB)
+|Elastic Load Balancing (ELB)
 |3
 |20 per region
-|By default, each cluster creates internal and external network load balancers for the primary API server and a single classic elastic load balancer for the router. Deploying more Kubernetes LoadBalancer Service objects will create additional link:https://aws.amazon.com/elasticloadbalancing/[load balancers].
+|By default, each cluster creates internal and external ELB Network Load Balancers for the primary API server and a single Classic Load Balancer for the router. Deploying more Kubernetes LoadBalancer Service objects will create additional link:https://aws.amazon.com/elasticloadbalancing/[load balancers].
```

Comment:
I think `each cluster creates internal and external Network Load Balancers for the primary API server ...` is OK, no need to mention ELB here.

Reply:
Thanks. Updated.